website/www/site/content/en/documentation/sdks/python-pipeline-dependencies.md

```diff
@@ -141,3 +141,12 @@ However, it may be possible to pre-build the SDK containers and perform the depe
 Dataflow, see [Pre-building the python SDK custom container image with extra dependencies](https://cloud.google.com/dataflow/docs/guides/using-custom-containers#prebuild).
 
 **NOTE**: This feature is available only for the `Dataflow Runner v2`.
+
+## Pickling and Managing Main Session
```

Comment:
If we're going to use Dataflow specific language here, we should specifically call that out in the section heading. I think this applies to other remote runners though.